_posts/2020-12-10-aws-fargate-serverless-deployments.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+authors: [yolan_vloeberghs]
+title: "Serverless Kubernetes deployments with AWS Fargate"
+image: /img/2020-12-10-aws-fargate-serverless-deployments.jpg
+tags: [AWS, Fargate, Kubernetes]
+category: Cloud
+comments: true
+---
+# Prerequisites
+ - An AWS account
+ - kubectl, aws-cli & eksctl installed
+ - Some Kubernetes knowledge
+
+# Introduction
+Introduced in late 2017, AWS Fargate is a compute engine for serverless container deployments. 
+The engine allows you to run containers without worrying about your infrastructure.   
+Node scaling and configuration is done by AWS, which means that you only have to worry about the health of your own resources, not of your Kubernetes infrastructure.  
+
+Fargate eliminates the struggle of configuring, scaling and initializing nodes on its own.
+Because as we know, managing clusters / servers / nodes can be a challenging and expensive task.
+
+Pricing is based on the CPU and RAM that your Pod allocates.
+You don't pay for separate nodes anymore, but instead per Pod which could drastically decrease computing costs.
```

Comment:
```suggestion
Instead of paying for compute nodes you pay for resources allocated to your pods which could drastically decrease computing costs.
```